public/templates/patientSignUp.html

```diff
@@ -61,92 +66,85 @@
-              <textarea
-                id="textArea"
-                name="paragraph_text"
-                placeholder="Please give a short description of your issue."
-                cols="50"
-                rows="10"
-                
-              ></textarea>
-
-              <button type="submit" id="button" class="button button-block" onclick ="formSubmit">
-<!--                 onclick="formSubmit" -->
-                <span style="color:#fff;">Match Me!</span>
-              </button>
-              <!--               <input
-                type="text"
-                placeholder="If you feel comfortable, please give a short description of what you're seeking help with."
-              /> -->
-            </div>
-
-            <!--           <div class="field-wrap">
-            <label>
-              Email Address<span class="req">*</span>
-            </label>
-            <input type="email"required autocomplete="off"/>
-          </div> -->
-          </form>
-        </div>
-
-        <div id="login">
-          <h1>Welcome Back!</h1>
-
-          <form action="/" method="post">
-            <div class="field-wrap">
-              <label> Email Address<span class="req">*</span> </label>
-              <input type="email" required autocomplete="off" />
+          <ul class="tab-group">
+            <li class="tab active"><a href="#physical">Physical Health</a></li>
+            <li class="tab"><a href="#mental">Mental Health</a></li>
+            <li class="tab"><a href="#emotional">Emotional Health</a></li>
+            <li class="tab"><a href="#talk">Spiritual Health</a></li>
+          </ul>
+          <!-- ***** Patient Topic Selection End ***** -->
+          
+          <!-- ***** Tab Content Start ***** -->
+          <div class="tab-content">
+
+            <!-- ***** Sign Up Area Start ***** -->
+            <div>
+              <h1 class="formText">Fill The Form To Get Matched!</h1>
+              <form action="/" method="post">
+                <!-- ***** Top Row Start ***** -->
+                <div class="top-row">
+                  <!-- Input field for first name -->
+                  <div class="field-wrap">
+                      <label>First Name<span class="req">*</span> </label>
+                    <input type="text" name="firstName" id="firstName"  autocomplete="off" required/>
+                  </div>
+                  <!-- Input field for last name -->
+                  <div class="field-wrap">
+                    <label>Last Name<span class="req">*</span> </label>
+                    <input type="text" name="lastName"  id="lastName"  autocomplete="off" required/>
+                  </div>
+                </div>
+                <!-- ***** Top Row End ***** -->
+                <!-- Input field for description of issue -->
+                <div class="field-wrap">
+                  <textarea
+                    id="textArea"
+                    name="paragraph_text"
```

Comment:
Based on the most recent convos, it seems like we are using the conventions found on Google Style Guide. I updated all the class/ID names to match this. Can I assume that this also applies to this name attribute? (I couldn't find anything about the name attribute on the style guide)
So instead of "paragraph_text", we use "paragraph-text"?

Reply:
I did this in my most recent commit, assuming the answer would be yes.